cores/esp8266/core_esp8266_main.cpp

```diff
@@ -156,11 +156,62 @@ void init_done() {
  * Peripherals (except for SPI0 and UART0) are not initialized.
  * This function does not return.
  */
+/*
+   A bit of explanation for this entry point:
+
+   SYS is the RTOS task/context used by the upperlying system to run its
```

Comment:
RTOS task/context -> SDK task context

There is confusion about RTOS and NONOS, I'd prefer to not add to that.